### PR TITLE
bean-config

### PR DESCRIPTION
**Pull request by OlexTul**
I propose such a class for Beans that are created from files.

### CODE REVIEW

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: maven
+      - name: Run Tests with Maven
+        run: mvn test --file pom.xml
```
Comment (git-alexbondarenko) [inline .github/workflows/build-and-test.yml]: tests run when build job executed, so this config will run tests twice

Comment (OlexTul) [reply .github/workflows/build-and-test.yml]: Thanks, I fixed it.

**Comment by OlexTul** [comment]
I used it to pull-request  #60

**Review by IfLunatic — APPROVED** [review]

**Comment by OlexTul** [comment]
Code used in SimpleDI https://github.com/GFLCourses5/worker/pull/97.

**Pull request closed without merging**